lib/init/config-initializer.js

```diff
@@ -438,10 +439,17 @@ function askInstallModules(modules, packageJsonExists) {
             result(input) {
                 return this.skipped ? null : input;
             }
+        },
+        {
+            type: "select",
+            name: "packageManager",
+            message: "Which package manager does your project use?",
+            initial: 0,
+            choices: ["npm", "yarn"]
         }
```

Comment:
Can we skip this step if the previous answer was "No"?

Also, I'm still not sure would it be better to have just one step with "No"/"Yes, with npm"/"Yes, with yarn".

If user doesn't want `npm` or `yarn` but some third package manager, at this point the only choice for them would be to somehow exit the process.

Reply:
> Can we skip this step if the previous answer was "No"?

Now skipped if `No` was selected. but we still have a different question for selecting the package manager.